controller/readonly.go

```diff
@@ -87,10 +87,14 @@ func (ctrl *ReadOnly) Run(ctx context.Context) error {
 
 func (ctrl *ReadOnly) checkInspect(ctx context.Context, u unit) (bool, error) {
 	taskName := u.taskName
+	d := u.driver
+	if !d.Task().IsEnabled() {
```

Comment:
Similar to a previous comment. Just wanted to make sure I understood if this change is it related to the thread-safe refactor or fixing something separately maybe?

Reply:
This one I had added because I thought it was missing after looking at the [`ReadWrite.checkApply` method](https://github.com/hashicorp/consul-terraform-sync/blob/master/controller/readwrite.go#L165-L178). After discussion it was added there to avoid event tracking. So it seems this is not needed here, will remove!

Reply:
Gotcha! Sorry for the confusion!